lasagne/tests/layers/test_conv.py

```diff
@@ -191,7 +197,7 @@ def test_defaults(self, Conv2DImpl, DummyInputLayer,
                 W=kernel,
                 **kwargs
             )
-            actual = layer.get_output_for(input).eval()
+            actual = get_output(layer, input).eval()
```

Comment:
oops, for some reason I was thinking it was using the deprecated method. 

I can change it back, but I should probably change `TestConv1DLayer` as well for consistency.